setup.py

```diff
@@ -51,15 +51,12 @@
 
 
 if sys.platform != 'win32':
-    compile_args = dict( extra_compile_args=['-O2', '-march=native',
-                                             '-Wno-unused-function',
-                                             '-Wno-unneeded-internal-declaration',
-                                             '-Wno-#warnings'],
+    compile_args = dict( extra_compile_args=['-O2', '-march=native'],
                              extra_link_args=['-O2', '-march=native'])
+    libraries = ["m"]
 else:
-    compile_args = dict( extra_compile_args=['-Wunused-function',
-                                              '-Wunneeded-internal-declaration',
-                                              '-Wno-#warnings'])
+    compile_args = dict( extra_compile_args=[])
+    libraries = []
```

Comment:
There is no such library with the MSVC compiler (adding it throws an error), the corresponding code is in the `msvcrt` (adding it doesn't help either), the [runtime library](https://msdn.microsoft.com/en-us/library/2kzt1wy3.aspx) which should be linked with the `\MD` flag (on by default). This indeed works with MSVC 2015 (PY3.5) but not MSVC 2008 (PY2.7) for some reason. Not sure I want to learn more details about the MSVC compiler regarding this )